utils/tests/searchresultnodate.go

```diff
@@ -0,0 +1,110 @@
+package tests
+
+import (
+	"github.com/jfrog/jfrog-cli-go/artifactory/commands/generic"
+	"github.com/jfrog/jfrog-cli-go/artifactory/spec"
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	clientutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+//Used in tests TestArtifactorySearchIncludeDir & 'TestArtifactorySearchProps'.
+//Inorder to compare hard coded json's struction result(e.g. GetSearchPropsStep)
+//witch cannot contain future 'Created' & 'Modified' dates
+type SearchResultNoDate struct {
+	Path  string              `json:"path,omitempty"`
+	Type  string              `json:"type,omitempty"`
+	Size  int64               `json:"size,omitempty"`
+	Props map[string][]string `json:"props,omitempty"`
+}
+
+type SearchCommandNoDate struct {
```

Comment:
There's probably no reason to duplicate the whole search command. It is enough to add a method on the *SearchResult* struct, which returns a new *SearchResultNoDate* struct.

Reply:
Removed 'SearchCommandNoDate' struct.
Created 'SearchResultNoDate' method instead.